### PR TITLE
Reset password and I18n

### PR DESCRIPTION
**Pull request by edudepetris**
the configurations to send email  are commented

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -3,6 +3,17 @@
...
+  # config.action_mailer.delivery_method = :smtp
+  # config.action_mailer.smtp_settings = {
+  # :enable_starttls_auto => true,
+  # :address => "smtp.gmail.com",
+  # :port => 587,
+  # :domain => "gmail.com",
+  # :authentication => :login,
+  # :user_name => "your-email example edu.depetris@gmail.com",
+  # :password => "your-password"
+  # }
```
Comment (fbrusatti) [inline config/environments/development.rb]: This is ok, but to do this in a perfect way we have to do something similar to what we did with database_example.yml file, this is proposse a file named `application_example.yml` and upload all keys defined on `application.yml` into a Constant in application.rb file.

So we save the task to define this to every environment, and each developer doesn't share any sensible information.

Check this [post](http://railscasts.com/episodes/85-yaml-configuration-revised?view=asciicast)

File: config/initializers/devise.rb [hunk 2]
```diff
@@ -10,11 +10,12 @@
   # ==> Mailer Configuration
   # Configure the e-mail address which will be shown in Devise::Mailer,
   # note that it will be overwritten if you use your own mailer class with default "from" parameter.
-  config.mailer_sender = "please-change-me-at-config-initializers-devise@example.com"
+  # reply-to:   movetequeentras@fulbo.com (example)
+  config.mailer_sender = "movetequeentras@fulbo.com"
```
Comment (fbrusatti) [inline config/initializers/devise.rb]: Exists the domain? and the email?

**Comment by fbrusatti** [comment]
@edudepetris is ok add a new file to a new locale, but you don't have to delete the existing locales (`devise.en.yml` have to be present too)

File: config/initializers/devise.rb (hunk 2 above)
Comment (edudepetris) [inline config/initializers/devise.rb]: no, it is only a example

Comment (fbrusatti) [inline config/initializers/devise.rb]: :+1:

**Comment by fbrusatti** [comment]
:baby_bottle:

**Pull request merged**